app/src/main/java/com/sdp13epfl2021/projmag/database/UserDataFirebase.kt

```diff
@@ -108,4 +109,19 @@ class UserDataFirebase(
             )
         }
     }
+
+    override fun pushCv(
+        cv: CurriculumVitae,
+        onSuccess: () -> Unit,
+        onFailure: (java.lang.Exception) -> Unit
+    ) {
+        getUserDoc()?.let {
+            doc ->
+            doc.set(
+                hashMapOf("cv" to cv),
```

Comment:
Using string literval inside the code could be avoided by using constants in the companion object,
where we can put something like that :
```
/**
 *  The field containing cv
 */
const val FAVORITES_FIELD = "cv"
```